mono/mini/genmdesc.c

```diff
@@ -18,13 +18,15 @@ void __nacl_suspend_thread_if_needed() {}
 
 #define MINI_OP(a,b,dest,src1,src2) b,
 #define MINI_OP3(a,b,dest,src1,src2,src3) b,
+#define MINI_OP_FLAG(a,b,dest,src1,src2,flags) b,
 /* keep in sync with the enum in mini.h */
 static const char* const
 opnames[] = {
 #include "mini-ops.h"
 };
 #undef MINI_OP
 #undef MINI_OP3
+#undef MINI_OP_FLAG
```

Comment:
Should be `MINI_OP3_FLAG` perhaps, since we'll likely need a `MINI_OP2_FLAG` in the future? Otherwise LGTM.